src/main/java/sirius/kernel/commons/AdvancedDateParser.java

```diff
@@ -730,42 +776,48 @@ private int fixYear(int year) {
     }
 
     /**
-     * Combines the parsed text along with the effective date (as <tt>Calendar</tt>).
+     * Combines the parsed text along with the effective date (as <tt>LocalDateTime</tt>).
      * <p>
      * The string representation of this contains the effective date in angular brackets. As there are ignored by
      * the parser, the resulting string can be re-parsed to refresh modifiers and computations.
      */
     public static class DateSelection {
 
-        private Temporal date;
-        private String dateString;
+        private final LocalDateTime date;
+        private final String dateString;
 
         /**
          * Creates a new <tt>DateSelection</tt> for the given calendar and input string.
          *
-         * @param calendar   the effective date to be used
+         * @param date       the effective date to be used
          * @param dateString the input string which yielded the given calendar
          */
-        DateSelection(Calendar calendar, String dateString) {
+        DateSelection(LocalDateTime date, String dateString) {
             super();
-            this.date = LocalDateTime.of(calendar.get(Calendar.YEAR),
-                                         calendar.get(Calendar.MONTH) + 1,
-                                         calendar.get(Calendar.DAY_OF_MONTH),
-                                         calendar.get(Calendar.HOUR_OF_DAY),
-                                         calendar.get(Calendar.MINUTE),
-                                         calendar.get(Calendar.SECOND));
+            this.date = date;
             this.dateString = dateString;
         }
 
         /**
          * Returns the effective date as <tt>Temporal</tt>
          *
          * @return the effective date. This might be <tt>null</tt> if parsing the expression failed.
+         * @deprecated use {@link #asDateTime()} which returns the proper type (<tt>LocalDateTime</tt>).
          */
+        @Deprecated
         public Temporal getTemporal() {
             return date;
         }
 
+        /**
+         * Returns the effective date as <tt>Temporal</tt>
+         *
+         * @return the effective date. This might be <tt>null</tt> if parsing the expression failed.
+         */
```

Comment:
Copy Pasta?
```suggestion
         * Returns the effective date as <tt>LocalDateTime</tt>
         *
         * @return the effective date. This might be <tt>null</tt> if parsing the expression failed.
         */
```